packages/rich-text/src/component/index.native.js

```diff
@@ -401,33 +412,66 @@ export class RichText extends Component {
 		this.lastAztecEventType = 'input';
 	}
 
-	handleMention( event ) {
+	handleTriggerKeyCodes( event ) {
 		const { keyCode } = event;
+		const triggeredOption = this.suggestionOptions().find( ( option ) => {
```

Comment:
Can the return from `this.suggestionOptions` be memorized to a class property so doesn't have to be called on every  key  code event?